src/randomField/UniformRandom.h

```diff
@@ -0,0 +1,42 @@
+
+#ifndef ARTSS_SOURCE_URANDOM_H_
+#define ARTSS_SOURCE_URANDOM_H_
+
+
+#include <random>
+#include <iostream>
+
+#include "../interfaces/IRandomField.h"
+
+
+class UniformRandom: public IRandomField {
+ public:
+    UniformRandom(real range, real step_size) :
+        m_seed(std::random_device()()),
+        m_steps(range / step_size) {
+        init_dist();
+    }
+
+    UniformRandom(real range, real step_size, int seed) :
+        m_seed(seed), m_steps(range / step_size) {
+        init_dist();
+    }
+
+    ~UniformRandom() = default;
+
+    Field random_field(size_t size) override;
+
+ private:
+     void init_dist() {
+         m_mt = std::mt19937(m_seed);
+         m_dist = std::uniform_int_distribution<int>(-m_steps, m_steps);
```

Comment:
shouldn't m_steps be also an integer if they are explicitly used for an integer distribution?

Reply:
of course, if we change it should be also changed in Functions.cpp but it would be better to replace Functions.cpp:Random if possible

Reply:
I totally agree with you. changed datatype to int in 7938fb8